src/main/java/redis/clients/jedis/Client.java

```diff
@@ -1507,6 +1507,12 @@ public void xrange(final String key, final StreamEntryID start, final StreamEntr
       SafeEncoder.encode(end == null ? "+" : end.toString()), count);
   }
 
+  @Override
+  public void xrange(String key, String start, String end, int count) {
```

Comment:
we should also support binary commands?
```
public void xrange(byte[] key, byte[] start, byte[] end, int count) {
```

Reply:
@yangbodong22011 those are already available in `Binary...` classes.

Reply:
@sazzad16 ohh...yes, ignore my comment